File: java/space-age/src/main/java/SpaceAge.java
class SpaceAge {

    private double sec;
    private static double ear = 31557600;

    SpaceAge(double seconds) {
        sec = seconds;
    }

    double getSeconds() {
        return sec;
    }

    double onEarth() {
        return sec/ear;
    }

    double onMercury() {
        return sec/(0.2408467 * ear);
    }

    double onVenus() {
        return sec/(0.61519726 * ear);
    }

    double onMars() {
        return sec/(1.8808158 * ear);
    }

    double onJupiter() {
        return sec/(11.862615 * ear);
    }

    double onSaturn() {
        return sec/(29.447498 * ear);
    }

    double onUranus() {
        return sec/(84.016846 * ear);
    }

    double onNeptune() {
        return sec/(164.79132 * ear);
    }

}

File: python/space-age/space_age.py
class SpaceAge:
    sec = 0
    ear = 31557600
    def __init__(self, seconds):
        self.sec = seconds
        pass
    def on_earth(self):   
        return round((self.sec/self.ear),2)

    def on_mercury(self):     
        return round(self.sec/(0.2408467 * self.ear),2)

    def on_venus(self):   
        return round(self.sec/(0.61519726 * self.ear),2)

    def on_mars(self):    
        return round(self.sec/(1.8808158 * self.ear),2)

    def on_jupiter(self):     
        return round(self.sec/(11.862615 * self.ear),2)

    def on_saturn(self):  
        return round(self.sec/(29.447498 * self.ear),2)

    def on_uranus(self):  
        return round(self.sec/(84.016846 * self.ear),2)

    def on_neptune(self):     
        return round(self.sec/(164.79132 * self.ear),2)
